bower.json

```diff
@@ -17,6 +17,7 @@
         "kubernetes-topology-graph": "0.0.23",
         "momentjs": "2.10.6",
         "mustache": "0.8.1",
+        "mutation-observer": "https://raw.githubusercontent.com/webmodules/mutation-observer/cdd0a54475a7d455dfd7de81d56748c80b65214b/index.js",
```

Comment:
Yes, bower couldn't find a version of this. I needed to add this to get a polyfill for MutationObsever on IE10, which we still support.